tests/integration/blockchain/test_integration_block_confirmations.py

```diff
@@ -6,7 +6,7 @@
 
 
 @pytest.mark.skip("This test need to be refactored to use some other transaction than deployment")
-def test_block_confirmations(testerchain, test_registry, mocker):
```

Comment:
Any reason the test itself needs to be renamed? Are there other tests with the same name?

Reply:
Not at all. I thought it was necessary, but I have just checked and renaming the file is enough.

✅ Done